Manage-Kubernetes-in-Google-Cloud.html

```
@@ -0,0 +1,1626 @@
+                        <div class="practice-title">② メトリックタイプは Counter を選ぶ</div>
+                        <p>
+                            今回は「エラーが何回発生したか」を数えたいだけなので、数値を抽出する
+                            Distribution 型ではなく、条件に一致したログ件数を単純にカウントする
+                            Counter 型を使います。Counter
+                            型はラベルが不要であればフィールドの追加設定なしでそのまま作成できるシンプルな構成です。
+                        </p>
+                    </div>
+
+                    <pre><code class="language-bash">gcloud logging metrics create pod-image-errors \
+  --description="Pod image reference errors" \
+  --log-filter='resource.type="k8s_container" AND severity>=ERROR'</code></pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)